share/exchange.yml

```diff
@@ -158,7 +158,11 @@ BSE:
   delay_amount: '10'
   display_name: Bombay Stock Exchange
   market_times:
-    partial_trading: {}
+    partial_trading:
+      dst_close: 13h
+      dst_open: 12h
+      standard_close: 14h
+      standard_open: 13h
     standard:
```

Comment:
Are you sure this information is correct ? The market usual trading hour is 03h45m to 10h00m. Your partial trading hour is 13h to 14h which mean this exchange will has late open and early close which sound wrong.